corehq/apps/locations/tests/test_bulk_management.py

```diff
@@ -1094,6 +1094,21 @@ def test_download_reupload_no_changes(self):
         self.assertFalse(save_location.called)
         self.assertFalse(save_type.called)
 
+    def test_dont_delete_referenced_location_types(self):
+        self.location_types['State'].expand_to = self.location_types['County']
+        self.location_types['State'].save()
+        delete_county_type = [
+            LocTypeRow('State', 'state', ''),
+            LocTypeRow('County', 'county', 'state', do_delete=True),
+            LocTypeRow('City', 'city', 'state'),
+        ]
+        result = self.bulk_update_locations(
+            delete_county_type,
+            [],
+        )
+        assert_errors(result, ["Location Type 'state' references the type to be deleted 'county'"
+                               " via the field 'expand_to'"])
```

Comment:
Trying to think how to make the error message more readable, maybe something like:

> "Cannot delete location type 'county', as it is referenced by location type 'state' via 'expand_to'

not sure how to work "expand_to" in there well, but whatever